go.mod

```diff
@@ -204,6 +204,9 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
+// Temporary replace until the next version of sqldb is taged.
+replace github.com/lightningnetwork/lnd/sqldb => ./sqldb
```

Comment:
If we merge this PR by itself first, then we can do the tag of this portion first, then merge the rest later.

Reply:
Perhaps best to merge this PR which contains @aakselrod's fixes, so the whole things goes in together.